pipx/main.py

```diff
@@ -488,6 +488,20 @@ def install(
     print(f"done! {stars}")
 
 
+def inject(venv_dir: Path, package: str, verbose: bool):
+    if not venv_dir.exists() or not next(venv_dir.iterdir()):
+        raise PipxError(f"Can't inject {package} into nonexistent venv {venv_dir}!")
+
+    venv = Venv(venv_dir, verbose=verbose)
+    venv.install_package(package)
+
+    if venv.get_package_version(package) is None:
+        raise PipxError(f"Could not find package {package}. Is the name correct?")
+
+    _list_installed_package(venv_dir)
```

Comment:
This should be removed since the listing will be unchanged.

Reply:
It's only there because I initially duplicated the `install` command. Removed!